doc/contributing/advocacy-ambasador-program.md

```diff
@@ -39,6 +39,15 @@ a discussion issue with the nomination in
 [nodejs/collaborators](https://github.com/nodejs/collaborators/)
 titled `Nomination X to be an Ambassador`, where X is the name of the person
 being nominated.
+If there is no objection from the TSC within 14 days, the nomination is approved.
+
+## Onboarding
+
+To onboard an ambassador, the a member of the TSC will:
```

Comment:
```suggestion
To onboard an ambassador, a member of the TSC will:
```

Reply:
The nomination can be done by every collaborator so a volunteer member of the tsc has to onboard just like regular nominations

Reply:
The suggestion is about fixing the English syntax error, not about changing the meaning of the sentence

Reply:
I misread the suggestion 🤦🏼